libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -68,8 +68,17 @@ void StaticAnalyzer::endVisit(FunctionDefinition const&)
 	m_nonPayablePublic = false;
 	m_constructor = false;
 	for (auto const& var: m_localVarUseCount)
-		if (var.second == 0)
-			m_errorReporter.warning(var.first->location(), "Unused local variable");
+		if (var.second == 0) {
```

Comment:
Curly bracket to go on the next line. (We should really put up a coding style and/or enforce clang-format :wink:)

Reply:
@axic I tested a clang-format configuration some time ago. It couldn't get it to handle a couple of cases in the current codebase (see this [StackOverflow question](https://stackoverflow.com/questions/44032087/clang-format-break-before-braces-for-lambda-functions-and-extern-blocks?noredirect=1#comment75842198_44032087)) but other than that, it worked perfectly.

Should I open a PR for it? (I remember the diff being big mostly due to different word wrapping, the rest was quite homogeneous in style).

Reply:
Yes please! I think @chfast may also help, because he uses it in another related project.

> I believe all of your BraceWrapping values are ignored if you don't have BreakBeforeBraces set to Custom.

Can you tried adjusting the format based on this comment?

Reply:
It is not possible to get the formatting with clang-format exactly like specified here and in cpp-ethereum. We might sacrifice some elements. And I think we should.

Reply:
This may open the discussion about our code formatting in general (might be good or bad).

Can you send a PR with the clang-format config?